cni/cmd/istio-cni-repair/main.go

```diff
@@ -204,3 +225,26 @@ func main() {
 		}
 	}
 }
+
+func setupMonitoring(addr, path string) {
+	mux := http.NewServeMux()
+	var listener net.Listener
+	if addr != "" {
```

Comment:
should we bypass all logic if address is empty? Also not sure if this check really makes sense since this is a private method and caller will never pass in empty address.